yarn-project/end-to-end/src/e2e_prover/full.test.ts

```diff
@@ -33,6 +34,7 @@ describe('full_prover', () => {
   it(
     'makes both public and private transfers',
     async () => {
+      RandomnessSingleton.getInstance().reseedIfDeterministic()
```

Comment:
Q: why is this needed?

Reply:
Hmm good question, there's a good argument for this going in setup() and 'makes both public and private transfers'